programmers/난이도별/level01.이상한_문자_만들기/develop-sell.js

```diff
@@ -0,0 +1,63 @@
+function solution(s) {
+  let arr = s.split(" ");
+  arr = arr.map((str)=> {
+      str = str.split("")
+      str = str.map((char, index) => {
+          return index % 2 === 0 || index === 0 ? char.toUpperCase() : char.toLowerCase()
+      })
```

Comment:
```suggestion
      str = str.map((char, index) => (
          index % 2 === 0 || index === 0 ? char.toUpperCase() : char.toLowerCase()
      ))
```

Reply:
```suggestion
      str = str.map((char, index) => index % 2 === 0 || index === 0 ? char.toUpperCase() : char.toLowerCase())
```

Reply:
```suggestion
      str = str.map((char, index) => (
          (index % 2 === 0 || index === 0) ? char.toUpperCase() : char.toLowerCase()
      ))
```
연산자 우선순위를 한눈에 보기 어려울 땐 괄호로 wrap 해주면 좋습니다